### ISSUE TITLE
Fail gracefully if JQ not installed.

### ISSUE THREAD

**Issue by KyleTryon** [CircleCI-Public/jira-connect-orb#10]
### Orb version
0.1.4
### What happened
A user utilized the JIRA orb on a custom Docker image without JQ installed and experienced an error.

```
Attempt to get Jira Issues from  https://circleci.com/api/v1.1/project/bitbucket/x/x/xxxx
/bin/bash: line 25: jq: command not found
Exited with code 127
```

### Expected behavior

For newer users, this message may be confusing for those that expect the integration to "just work". A better solution may be to add a check for required dependencies and gracefully error with instructions to resolve.

**Comment by iynere** [comment]
why don't we just install JQ if it's not available ? there's a whole orb for that:

https://circleci.com/orbs/registry/orb/circleci/jq

**Comment by iynere** [comment]
patched in `circleci/jira@1.0.5`
